keps/sig-cli/2906-kustomize-function-catalog/README.md

```diff
@@ -743,30 +788,32 @@ definitions:
         type: array
         items:
           type: string
-  KRMFunction:
+  KRMFunctionDefinition:
```

Comment:
We need to make the same changes to https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/2985-public-krm-functions-registry, right? It leverages the `KRMFunction` definition from here.

Reply:
We will need to update it if we change the name here.

Reply:
Good call--the `names.kind` change needs to be made there as well.